.github/workflows/build-test.yml

```diff
@@ -142,25 +146,64 @@ jobs:
       - name: Check formatting with ESLint
         run: yarn lint
 
+  dismiss-previous-e2e-check:
```

Comment:
we already have `cancel-in-progress: true` on L24, what is this specifically for?